multicluster/controllers/multicluster/serviceexport_controller.go

```diff
@@ -313,15 +348,18 @@ func (r *ServiceExportReconciler) updateSvcExportStatus(ctx context.Context, req
 	now := metav1.Now()
 	var res, message *string
 	switch cause {
-	case notFound:
+	case serviceNotFound:
 		res = getStringPointer("not_found_service")
```

Comment:
We should change to "service_not_found".

Reply:
Done